bears/r/FormatRBear.py

```diff
@@ -20,12 +20,12 @@ class FormatRBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         r_keep_comments: bool=True,
+                         r_keep_comments: bool=None,
                          r_keep_blank_lines: bool=True,
                          r_braces_on_next_line: bool=False,
-                         r_use_arrows: bool=False,
+                         r_use_arrows: bool=None,
                          tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
-                         r_max_expression_length: int=20):
+                         r_max_expression_length: int=0):
```

Comment:
actually theres is no default value for this one, it must be passed to
`tidy_source` explicitly and its value varies from 20 to 500 :) should i
mention this somewhere? :)

2016-06-18 19:49 GMT+01:00 Mischa Krüger <notifications@github.com>:

> In bears/r/FormatRBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/525#discussion_r67604948>
> :
>
> >                           tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
> > -                         r_max_expression_length: int=20):
> > +                         r_max_expression_length: int=0):
>
> hm seems git is right. Wanna do another commit that sets the default?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/525/files/d84aca049bf7210b89f225a17fcc69289c4832c8#r67604948>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xojh8ICEr4n_RhtXlTggQLrEbBz6Bks5qND3TgaJpZM4I14vI>
> .
>